client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -159,20 +171,22 @@ float disp_z;
 #if defined(ENABLE_DIRECTIONAL_SHADING) && !LIGHT_EMISSIVE
 	// Lighting color
 	vec3 resultLightColor = ((lightColor.rgb * gl_Color.a) + nightRatio);
+	resultLightColor = from_sRGB_vec(resultLightColor);
 
 	// ((resultLightColor * ((max(dot(normal, lightDirection), -0.2) + 0.2) / 1.2)* 0.6)) + 0.4;
-	float ambient_light = to_sRGB(0.15);
+	float ambient_light = 0.3;
 	float directional_light = dot(alwaysNormal, lightDirection);
 	directional_light = max(directional_light + 0.2, 0.0);
-	directional_light *= 0.5;
-	resultLightColor = resultLightColor * to_sRGB(directional_light) + ambient_light;
+	directional_light *= (1.0 - ambient_light) / 1.2;
+	resultLightColor = resultLightColor * directional_light + ambient_light;
 
 	directional_light = dot(alwaysNormal, artificialLightDirection);
 	directional_light = max(directional_light + 0.5, 0.0);
-	directional_light *= 0.47;
-	float artificialLightShading = to_sRGB(directional_light + 0.3);
+	directional_light *= (1.0 - 0.3) / 1.5;
```

Comment:
How often is this part called? Multiplying by its inverse might give a slight performance improvement on weaker GPUs.

Reply:
I haven't added these changes to the other vertex shaders; this still needs testing.